docs/BUILD

```diff
@@ -9,20 +9,23 @@ exports_files(["protodoc_manifest.yaml"])
 
 envoy_package()
 
-# TODO(phlax): fix failing/excluded configs
-# the following config only fails on windows:
-#       dns-cache-circuit-breaker: "Error: unable to read file: /etc/ssl/certs/ca-certificates.crt"
-
 filegroup(
     name = "configs",
     srcs = glob(
-        ["root/**/*.yaml"],
+        [
+            "root/**/*.yaml",
+            "root/**/*.pb",
+        ],
         exclude = [
-            "root/intro/_include/life-of-a-request.yaml",
+            # windows: "Error: unable to read file: /etc/ssl/certs/ca-certificates.crt"
+            "root/configuration/http/http_filters/_include/dns-cache-circuit-breaker.yaml",
             "root/intro/arch_overview/security/_include/ssl.yaml",
```

Comment:
Can we make this a TODO and work with @envoyproxy/windows-dev to get this fixed?